README.md

```diff
@@ -11,27 +11,6 @@
 - CSS
 
 
-## Getting Started
-
-To start to work with this repository, Clone it in your personal computer using git clone.
-
-
-To get a local copy up and running follow these simple example steps.
-
```

Comment:
- [ ] Nice job with your README file , you add most of the session needed but please add section about Getting starting to show the different steps on how to start to work with this repository .

Reply:
Ok, thank you very much for all your observations. I already fixed this point.